extensions/consoledev/package.json

```
@@ -0,0 +1,53 @@
+{
+  "name": "consoledev",
+  "title": "Console Dev",
+  "description": "An extension to discover the best tools and beta releases for developers (via console.dev)",
+  "icon": "console-icon.png",
+  "author": "fedevitaledev",
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "tools_list",
+      "title": "Tools List",
+      "subtitle": "Console - interresting developers tools",
```

I assume `interresting` should be `interesting` here?
```suggestion
      "subtitle": "Console - interesting developers tools",
```